beacon-chain/p2p/subnets.go

```diff
@@ -15,8 +17,15 @@ import (
 )
 
 var attestationSubnetCount = params.BeaconNetworkConfig().AttestationSubnetCount
+var syncCommsSubnetCount = params.BeaconConfig().SyncCommitteeSubnetCount
 
 var attSubnetEnrKey = params.BeaconNetworkConfig().AttSubnetKey
+var syncCommsSubnetEnrKey = params.BeaconNetworkConfig().SyncCommsSubnetKey
+
+// The value used with the subnet, inorder
+// to create an appropriate key tor retrieve
+// the relevant lock.
+const syncLockerVal = 100
```

Comment:
This seems arbitrary...it is arbitrary right? Maybe some more clarification about this choice of number, even if it was random, would be helpful